docs/self-managed/components/orchestration-cluster/core-settings/concepts/backups.md

```diff
@@ -0,0 +1,27 @@
+---
+id: backups
+title: "Backups"
+description: "Learn more about backups with the Orchestration Cluster."
+---
+
+When running an orchestration cluster with **Operate** and **Tasklist**, you must configure a snapshot repository in your chosen database:
+
+- [Elasticsearch snapshot repository](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html)
+- [OpenSearch snapshot repository](https://docs.opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/)
+
+Both Operate and Tasklist are configured with the snapshot repository name to trigger database snapshots. This ensures coherent backups across all components.
+
+:::info
+Learn more about the backup procedure and why it must be triggered through Camunda components in the [backup guide](/self-managed/operational-guides/backup-restore/backup-and-restore.md).
+:::
+
+## Configuration parameters
+
+| Component | Configuration key                        | Description                      | Default value |
+| --------- | ---------------------------------------- | -------------------------------- | ------------- |
+| Operate   | `camunda.operate.backup.repositoryName`  | ES / OS snapshot repository name | -             |
```

Comment:
This is a legacy configuration key.
The new configuration key is `camunda.data.backup.repository-name`.
https://docs.google.com/spreadsheets/d/1R4epsy6DWemA8_76cUE6zOGUFbrXCXlM2reXnKFuz7Y/edit?gid=818158292#gid=818158292&range=149:149

Is this intended? Maybe I'm missing context

Reply:
Adjusting in latest commit 👍